src/serial.c

```diff
@@ -0,0 +1,115 @@
+    #include "io.h" /* io.h is implement in the section "Moving the cursor" */
```

Comment:
Please fix indentation here.

Reply:
`/* io.h is implement in the section "Moving the cursor" */` - comment is unnecessary